source/site-administrators/studio/rte2-setup.rst

```diff
@@ -148,6 +148,107 @@ You can also add custom elements to the rule set and can be done by simply addin
       :width: 85%
       :align: center
 
+   |
+
+.. _adding-external-plugins:
+
+^^^^^^^^^^^^^^^^^^^^^^^
+Adding External Plugins
+^^^^^^^^^^^^^^^^^^^^^^^
+
+TinyMCE provides an option to specify URLS to plugins outside the tinymce plugins directory.  These external plugins allows the user to extend TinyMCE.  For example, you can create custom dialogs, buttons, menu items, etc.
```

Comment:
```suggestion
TinyMCE provides an option to specify URLS to plugins outside the tinymce plugins directory. These external plugins allow the user to extend TinyMCE. For example, you can create custom dialogs, buttons, menu items, etc.
```

- A link & reference to the relevant TinyMCE docs would be good, I think. (Don't if we have a policy for including these sort of links on our docs)
  - https://www.tiny.cloud/docs/configure/editor-appearance/#toolbarn
  - https://www.tiny.cloud/docs/configure/integration-and-setup/#external_plugins

- We should mention that adding a custom button to the toolbar in Crafter would be done using the plugin route since, [what TinyMCE docs advise](https://www.tiny.cloud/docs/demo/custom-toolbar-button/) is not viable on our system without doing a whole form control plugin where they'd have full control of tinymce initialization. Plugins can do anything but in our world, it is also the way to add toolbar items (that aren't OOTB buttons, of course).

- The example below shows how to do just this, so perhaps you can tie the two together (i.e. these intro with the example below).